### PR TITLE
include addition of app whitelisting support

### PR DESCRIPTION
**Pull request by SteveL-MSFT**
Version(s) of document impacted
------------------------------
- [x] Impacts 6.1 document
- [ ] Impacts 6.0 document
- [ ] Impacts 5.1 document
- [ ] Impacts 5.0 document
- [ ] Impacts 4.0 document
- [ ] Impacts 3.0 document

Reason(s) for not updating all version of documents
--------------------------------------------------
- [x] The documented feature was introduced in version (list version here) of PowerShell
- [ ] This issue only shows up in version (list version(s) here) of the document
- [ ] This PR partially fixes the issue, and issue #<insert here> tracks the remaining work

### CODE REVIEW

File: reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-61.md [hunk 1]
```diff
@@ -37,6 +37,12 @@ so that any modules or scripts that use these APIs can rely on them being availa
 
 The Windows Compatibility Pack enables PowerShell Core to use **more than 1900 cmdlets that ship with Windows 10 October 2018 Update and Windows Server 2019**.
 
+## Support for Application Whitelisting
+
+PowerShell Core 6.1 has parity with Windows PowerShell 5.1 supporting [AppLocker](https://docs.microsoft.com/en-us/windows/security/threat-protection/windows-defender-application-control/applocker/applocker-overview)
```
Comment (PaulHigin) [inline reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-61.md]: "PowerShell Core 6.1 has parity with Windows PowerShell 5.1"
Actually this isn't quite true.  PSCore will not have parity with Windows PowerShell until the merge work is completed.  I think it is more accurate to say that this release of PSCore has added application white listing support for Windows platform.

File: reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-61.md [hunk 2]
```diff
@@ -37,6 +37,12 @@ so that any modules or scripts that use these APIs can rely on them being availa
...
+and [Device Guard](https://docs.microsoft.com/en-us/windows/security/threat-protection/device-guard/introduction-to-device-guard-virtualization-based-security-and-windows-defender-application-control) application whitelisting.
```
Comment (PaulHigin) [inline reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-61.md]: "DeviceGuard has been re-branded to "Windows Defender Application Control".  This is annoying but I think we should mention both titles.

File: reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-61.md [hunk 3]
```diff
@@ -37,6 +37,12 @@ so that any modules or scripts that use these APIs can rely on them being availa
...
+Application whitelisting allows granular control of what binaries are allowed to be executed used with PowerShell [Constrained Language mode](https://blogs.msdn.microsoft.com/powershell/2017/11/02/powershell-constrained-language-mode/).
```
Comment (PaulHigin) [inline reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-61.md]: Perhaps:
Application white listing provides granular control of which binaries are allowed to run on a system.  PowerShell runs in Constrained Language mode when such a policy is enforced on a system, and only allows policy trusted script files to run in Full Language mode.

File: reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-61.md (hunk 1 above)
Comment (SteveL-MSFT) [reply reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-61.md]: I distinctly called out feature parity (specific to Lee's feature table).

File: reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-61.md (hunk 2 above)
Comment (SteveL-MSFT) [reply reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-61.md]: Looks like this was merged.  Please submit new PR, your feedback makes sense.

File: reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-61.md (hunk 3 above)
Comment (SteveL-MSFT) [reply reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-61.md]: Looks like this was merged. Please submit new PR, your feedback makes sense.

**Pull request merged**